packages/dialog/test/draggable-resizable.test.js

```diff
@@ -566,14 +562,17 @@ describe('draggable', () => {
 
   it('should not reset scroll position on dragstart', async () => {
     dialog.modeless = true;
-    button.style.marginBottom = '200px';
-    dialog.$.overlay.setBounds({ height: '100px' });
-    await nextUpdate(dialog);
-    // TODO use dialog.$.overlay.$.content.scrollTop with base styles
-    container.scrollTop = 100;
-    expect(container.scrollTop).to.equal(100);
+    dialog.height = '100px';
+    await nextRender();
+
+    const contentElement = dialog.firstElementChild;
+    contentElement.style.minHeight = '200px';
+    await nextResize(contentElement);
```

Comment:
Setting margin on the button doesn't trigger content resize. Changed to use `minHeight` explicitly.